opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -31,35 +31,64 @@
 import os
 import sys
 import opentimelineio as otio
+from collections import Iterable
 
 lib_path = os.environ.get("OTIO_AAF_PYTHON_LIB")
 if lib_path and lib_path not in sys.path:
     sys.path += [lib_path]
 
-import aaf  # noqa (E402 module level import not at top of file)
-import aaf.storage  # noqa
-import aaf.mob  # noqa
-import aaf.define  # noqa
-import aaf.component  # noqa
-import aaf.base  # noqa
+import aaf2
+import aaf2.content
+import aaf2.mobs
+import aaf2.components
+import aaf2.core
 
 debug = False
 __names = set()
 
 
+def _get_parameter(item, parameter_name):
+    values = dict([(value.name, value) for value in item.parameters.value])
```

Comment:
fwiw I don't think you need the [ ], this should work too:
```suggestion
    values = dict((value.name, value) for value in item.parameters.value)
```